src/app/styles/components/_preview-card.scss

```diff
@@ -55,6 +56,7 @@
         flex-flow: column nowrap;
         align-items: stretch;
         justify-content: space-between;
+        margin-bottom: 5rem;
```

Comment:
Not sure how SASS handles mixed units, but you can totally drop those mixed units in a CSS `calc()` function and it should _just work_. 

Something like:
`margin-bottom: calc(#{oto-more-posts-btn-height} + 2 * #{$home-section-padding});`